### PR TITLE
Fix avatarUrl helper, add tests

### PR DESCRIPTION
**Pull request by francocorreasosa**

### CODE REVIEW

File: core/components/_helpers/avatar-url.js [hunk 1]
```diff
@@ -9,6 +9,9 @@ const urlTemplate = (hash, initials, size) =>
   `https://s.gravatar.com/avatar/${hash}?s=${size}&r=pg&d=${initialsAvatar(initials)}`
 
 export default function avatarUrl(email, initials, size = 480) {
-  const hash = md5(email.toLowerCase())
+  if (!email && !initials)
+    throw new Error('You need to provide either `email`, `initials` or both to the Avatar.')
```
Comment (cristiandouce) [inline core/components/_helpers/avatar-url.js]: isn't that why we do have a default profile svg?

File: internal/test/unit/helpers/avatar-url.test.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      'https://s.gravatar.com/avatar/23463b99b62a72f26ed677cc556c44e8?s=480&r=pg&d=https%3A%2F%2Fcdn.auth0.com%2Favatars%2Ffc.png'
+    )
+  })
+
+  it('to return a custom size URL if size is provided', () => {
+    const url = avatarUrl('example@example.com', 'FC', 368)
+    expect(url).toEqual(
+      'https://s.gravatar.com/avatar/23463b99b62a72f26ed677cc556c44e8?s=368&r=pg&d=https%3A%2F%2Fcdn.auth0.com%2Favatars%2Ffc.png'
+    )
+  })
```
Comment (cristiandouce) [inline internal/test/unit/helpers/avatar-url.test.js]: ❤️

File: core/components/_helpers/avatar-url.js (hunk 1 above)
Comment (francocorreasosa) [reply core/components/_helpers/avatar-url.js]: That case is handled by the Avatar component itself (by rendering a local SVG, so not a URL).

Comment (francocorreasosa) [reply core/components/_helpers/avatar-url.js]: Nope, just checked. This is a CDN URL, let me change this so we can centralize it :)

Comment (francocorreasosa) [reply core/components/_helpers/avatar-url.js]: Done

**Pull request merged**